src/ducks/Signals/TriggerForm.module.scss

```diff
@@ -117,6 +137,60 @@ label {
   }
 }
 
+.TriggerFormItem {
+  border: 1px solid #e7eaf3;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin-bottom: 20px;
+}
+
+.triggerHeader {
+  width: 100%;
+  background: #f9fafc;
+  border: 1px solid #e7eaf3;
```

Comment:
Same comment as a`[DM_2_1]`. Please, take a look at modified `.scss` files for such cases